exercises/critical-path.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
I just realize that this is actually already here under "make-a-map.md"

Reply:
Oh, good catch. That name is strange to me. All of the content on that page references "Critical Path" how would you feel about renaming "Make a Map" to "Critical Path"?

Reply:
Thats ok with me! Maybe lets remove this one from this PR and do the rename in a seperate commit?

Reply:
For sure

Reply:
FYI, I already changed the rack_rewrite for thoughtbot.com to the new name